capability.go

```diff
@@ -381,6 +388,65 @@ func (c Client) SendCall(ctx context.Context, s Send) (*Answer, ReleaseFunc) {
 	return ans, rel
 }
 
+// SendStreamCall is like SendCall except that:
+//
+// 1. It does not return an answer for the eventual result.
+// 2. If the call returns an error, all future calls on this
+//    client will return the same error (without starting
+//    the method or calling PlaceArgs).
+func (c Client) SendStreamCall(ctx context.Context, s Send) error {
+	var (
+		streamError error
+		wg          *sync.WaitGroup
+	)
+	syncutil.With(&c.mu, func() {
+		streamError = c.streamError
+		if streamError == nil {
+			if c.streamWg == nil {
+				c.streamWg = &sync.WaitGroup{}
```

Comment:
Is there a reason for this lazy initialization, as opposed to removing the pointer indirection on `streamWg`?